runtime/sema/check_dictionary_expression.go

```diff
@@ -106,7 +111,7 @@ func IsValidDictionaryKeyType(keyType Type) bool {
 		return keyType.Kind == common.CompositeKindEnum
 	default:
 		switch keyType {
-		case NeverType, BoolType, CharacterType, StringType, MetaType:
+		case NeverType, BoolType, CharacterType, StringType, MetaType, InvalidType:
```

Comment:
Why is this needed here? Is the idea here that we allow the invalid type to avoid the additional error that the type is not a valid key type, as there was already an error report for the invalid type?

Reply:
ah, I had added this initially, but later moved this check to the `VisitDictionaryExpression` method itself. This is a leftover that I should have cleaned.

Reply:
Cleaned it up in 67e3268522d210d752cc49c0a34613fda665826b